cmd/submit_relative_path_windows_test.go

```diff
@@ -55,5 +56,5 @@ func TestSubmitRelativePath(t *testing.T) {
 	assert.NoError(t, err)
 
 	assert.Equal(t, 1, len(submittedFiles))
-	assert.Equal(t, "This is a file.", submittedFiles["/file.txt"])
+	assert.Equal(t, "This is a file.", submittedFiles["\\file.txt"])
```

Comment:
OMG. Of course.

This should be `string(os.PathSeparator) + "file.txt"` or something like that. I wonder if that will automatically escape the string.

Reply:
I don't think it will escape, but escaping is not needed as the key actually is `\file.txt`. Changing to `string(os.PathSeparator) + "file.txt"` works as expected. However, I have a question why does the filename need to have the path separator? Should it not just be the filename?

Reply:
It's what the exercism API sends back, relative to the workspace directory, including the path separator. We could have done it either way, I don't think we had any reason to do one over the other.

Reply:
s/sends back/expects/ (sends back on the download)

Reply:
Wait. uhm.... is it just setup in the test that does this? In that case I think you're right, we should just remove it.

Reply:
I think the most reasonable thing right now is to keep the leading slash, since there are so many moving parts. It could be worth reconsidering down the road, as whether we change this now or later, we still have to be able to take into account the exercises that _do_ have the leading slash.

Reply:
I agree with you on this one; especially since I have not been able to give a lot of time. I would hate to start a larger thing right now and not be able to follow it all the way through. 

If you are okay with it I will merge this in.

Reply:
Yepp, I think that's fine. I'll go ahead and merge.